data/campaigns/The_South_Guard/scenarios/01_Born_to_the_Banner.cfg

```diff
@@ -584,19 +584,15 @@
 
         [message]
             speaker=Aleron
-            message= _ "Not more bandits... We have nothing left. Begone!"
+            message= _ "More bandits? Begone from here, we have nothing left to give you."
         [/message]
         [message]
             speaker=unit
-            message= _ "We are no bandits. The banner of the South Guard flies again! Help us drive these criminals from your land."
+            message= _ "We fight under the banner of the South Guard <i>against</i> the bandits! You should join us if you wish to drive these brigands from your land!"
```

Comment:
No correction here, just wanted to express my joy at seeing "brigands". Underused word.